components/NavigationBar/tests/unit/NavigationBarTests.m

```diff
@@ -199,6 +200,32 @@ - (void)testTitleFontProperty {
   XCTAssertEqual(fontTraits, resultTraits);
 }
 
+- (void)testTitleFontPropertyWithAllowAnyTitleFontSizeEnabled {
+  MDCNavigationBar *navBar = [[MDCNavigationBar alloc] init];
```

Comment:
Please add the Given-When-Then comments to define the different sections of the test.

Reply:
Done.